service/src/main/java/bio/terra/workspace/service/job/JobService.java

```diff
@@ -465,6 +395,23 @@ public void setFlightDebugInfoForTest(FlightDebugInfo flightDebugInfo) {
     this.flightDebugInfo = flightDebugInfo;
   }
 
+  private void logAlert(String msg, String flightId) {
```

Comment:
there's also a `LoggingUtils.logAlert` method that might be useful here. It only takes a String so you'd need to format/concat the flight ID into the message, but it saves a bit of duplication

Reply:
That is a good idea. Since I am out of time, I'm going to merge as is and clean that up in another PR.